pkgs/top-level/all-packages.nix

```diff
@@ -153,6 +153,8 @@ in
 
   buildFHSUserEnv = callPackage ../build-support/build-fhs-userenv { };
 
+  buildFHSUserEnvBw = callPackage ../build-support/build-fhs-userenv-bw { };
```

Comment:
If we are going for a new name, maybe we can find something that sounds less obscure.
I feel like adding a `Bw` suffix makes it a bit worse. On the other hand we could also just introduce this a new version of buildFHSUserEnv i.e. buildFHSUserEnv_2. At some point we should drop the old version.

Reply:
As I said in the other PR, the name is supposed to be temporary. Once everything is ported, we would drop the suffix entirely.

Reply:
Ok. I would still prefer a version number in this case to clearly communicate that.

Reply:
I've made it clearer what is what now.

I also changed chrootenv -> fhsenv where applicable.

Reply:
lgtm.